account-kit/smart-contracts/src/ma-v2/client/client.test.ts

```diff
@@ -0,0 +1,69 @@
+import { custom, parseEther, publicActions } from "viem";
+
+import { LocalAccountSigner, type SmartAccountSigner } from "@aa-sdk/core";
+
+import { createSMAV2AccountClient } from "./client.js";
+
+import { local070Instance } from "~test/instances.js";
+import { setBalance } from "viem/actions";
+import { accounts } from "~test/constants.js";
+
+describe("MA v2 Tests", async () => {
+  const instance = local070Instance;
+  let client: ReturnType<typeof instance.getClient> &
+    ReturnType<typeof publicActions>;
+
+  beforeAll(async () => {
+    client = instance.getClient().extend(publicActions);
+  });
+
+  const signer: SmartAccountSigner = new LocalAccountSigner(
+    accounts.fundedAccountOwner
+  );
+
+  it("sends a simple UO", async () => {
+    const provider = await givenConnectedProvider({ signer });
+
+    await setBalance(instance.getClient(), {
+      address: provider.getAddress(),
+      value: parseEther("2"),
+    });
+
+    const target = "0x000000000000000000000000000000000000dEaD";
+    const sendAmount = parseEther("1");
+
+    const startingAddressBalance = await client.getBalance({
+      address: target,
+    });
+
+    const result = await provider.sendUserOperation({
+      uo: {
+        target: target,
+        value: sendAmount,
+        data: "0x",
+      },
+    });
+
+    const txnHash1 = provider.waitForUserOperationTransaction(result);
+    await expect(txnHash1).resolves.not.toThrowError();
```

Comment:
Nit: you could simplify these lines as just
```ts
await provider.waitForUserOperationTransaction(result);
```
which will already fail the text if it results in an error.